### PR TITLE
Update fpauth-strings.xml

### PR DESCRIPTION
**Pull request by Fever905**
Changed Fingerprint Authentication title to Fingerprint Secured.  Previous title does not fit on screen.

### CODE REVIEW

**Comment by Fever905** [comment]
Sorry if I did something wrong guys, this is my first attempt at a fork / pull-request.

**Comment by NiklasMerz** [comment]
Your pull request is fine. Could you please give some information about the issue you are fixing? Which device are you referering to? Maybe we could find another title, since I think "secured" ist not quite fitting.

I looking forward to hear which device you tested. Thanks for your contribution. Hey it´s [Hacktober](https://hacktoberfest.digitalocean.com/). 👍

**Comment by Fever905** [comment]
Nexus 6P in portrait mode.  Should I post a screen shot?  I'm very new to this but I really want to learn. Is there a way for me to specify my own version and include this in my app build?  (Using Phonegap Build).
Is there a reason why you can't specify the titles a configu variable?  Is it because those components need to be compiled before they can run.. Makes sense if so.

**Comment by NiklasMerz** [comment]
Sounds strange to me that such a big device has problems with this title. You are right. These strings which are part of the xml file you changed are includes at build time and used in the native dialog. Changing the title to be configurable via Javascript is possible.

If you want to include your customized plugin to your app, just use your github url instead of the npm package. If you have further questions you can also talk to me on gitter.

File: package.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
   },
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/niklasmerz/cordova-plugin-fingerprint-aio"
+    "url": "git+https://github.com/Fever905/cordova-plugin-fingerprint-aio"
```
Comment (NiklasMerz) [inline package.json]: This is not necessary for building your app with an custom version and should not be part of the pull request

**Comment by NiklasMerz** [comment]
@Fever905 I am closing this for now. If you still think this should be changed please look at the review and reopen this again.
